apps/pie-storybook/stories/contribution/overview.mdx

```diff
@@ -0,0 +1,220 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Contribution/Overview" />
+
+# PIE web components - Contribution guide
+
+This guide walks you through the full contribution process for the PIE web component library, from setup to pull request.
+
+## Table of Contents
+
+1. [Get involved](#1-get-involved)
+2. [Machine setup](#2-machine-setup)
+3. [Clone the repository](#3-clone-the-repository)
+4. [Scaffold a new component](#4-scaffold-a-new-component)
+5. [Develop and test the component](#5-develop-and-test-the-component)
+6. [Component package anatomy](#6-component-package-anatomy)
+7. [Testing standards](#7-testing-standards)
+8. [Create a pull request](#8-create-a-pull-request)
+9. [Add a changeset (if needed)](#9-add-a-changeset-if-needed)
+10. [Trigger a snapshot release](#10-trigger-a-snapshot-release)
+11. [References](#11-references)
+
+## 1. Get involved
+
+Before starting a new component, please contact the Design System team. They will help define requirements and ensure the component is suitable for the shared library.
+
+## 2. Machine setup
+
+Set up your development environment for the PIE monorepo.
+
+### Required tools
+
+- Install a code editor ([VSCode](https://code.visualstudio.com/) recommended)
+- (Optional) Install the [PIE VSCode extension](https://marketplace.visualstudio.com/items?itemName=JamieMaguire.pie-design-system-vscode-prototype)
+- Install [Git secrets](https://github.com/awslabs/git-secrets)
+- Install [Volta](https://docs.volta.sh/guide/getting-started) (manages Node/Yarn versions):
+
+### Environment Variables
+
+Request these from the Design System team:
+
+- `TURBO_TOKEN`
+- `PERCY_TOKEN_{COMPONENT}` (e.g. `PERCY_TOKEN_PIE_BUTTON`)
+
+Add them to your environment:
+
+> How to best add these to your environment will depend on your operating system. If in doubt, feel free to reach out to the Design System team for help!
+
+```bash
+# .zshrc or .bash_profile
+export TURBO_TOKEN=your_token
+export PERCY_TOKEN_PIE_BUTTON=your_token
+```
+
+## 3. Clone the repository
+
+Start by cloning the [PIE monorepo](https://github.com/justeattakeaway/pie) and installing dependencies via `yarn install`.
+
+> Make sure you're on the `main` branch and have synced the latest changes before starting new work.
+
+## 4. Scaffold a new component
+
+Use our [Yeoman generator](https://github.com/justeattakeaway/pie/tree/main/packages/tools/generator-pie-component) to create a new component.
+
+Follow the README for usage.
+
+## 5. Develop and test the component
+
+### Start storybook
+
+```bash
+yarn dev --filter=@justeattakeaway/pie-storybook
+```
+
+### Watch the component for changes
+
+In another window whilst Storybook is being served:
+
+```bash
+yarn watch --filter=@justeattakeaway/pie-{component-name}
+```
+
+### Build the component
+
+```bash
+yarn build --filter=@justeattakeaway/pie-{component-name}
+```
```

Comment:
Is it worth mentioning that if you're watching the component above, you don't need to also build the component, as that happens automatically on save?